tket/src/serialize/pytket/extension/prelude.rs

```diff
@@ -1,15 +1,20 @@
 //! Encoder and decoder for tket operations with native pytket counterparts.
 
+use std::sync::Arc;
+
 use super::PytketEmitter;
 use crate::serialize::pytket::config::TypeTranslatorSet;
+use crate::serialize::pytket::decoder::{
+    DecodeStatus, LoadedParameter, PytketDecoderContext, TrackedBit, TrackedQubit,
+};
 use crate::serialize::pytket::encoder::{EncodeStatus, PytketEncoderContext};
-use crate::serialize::pytket::extension::{PytketTypeTranslator, RegisterCount};
-use crate::serialize::pytket::PytketEncodeError;
+use crate::serialize::pytket::extension::{PytketDecoder, PytketTypeTranslator, RegisterCount};
+use crate::serialize::pytket::{PytketDecodeError, PytketEncodeError};
 use crate::Circuit;
-use hugr::extension::prelude::{BarrierDef, TupleOpDef, PRELUDE_ID};
+use hugr::extension::prelude::{bool_t, qb_t, BarrierDef, Noop, TupleOpDef, PRELUDE_ID};
 use hugr::extension::simple_op::MakeExtensionOp;
 use hugr::extension::ExtensionId;
-use hugr::ops::ExtensionOp;
+use hugr::ops::{ExtensionOp, OpType};
 use hugr::types::TypeArg;
 use hugr::HugrView;
 use tket_json_rs::optype::OpType as Tk1OpType;
```

Comment:
Rename to `PytketOpType`?

Reply:
ok got it, but this would just be a file-local rename... You can probably do it 🙃